Libs/DICOM/Widgets/ctkDICOMBrowser.cpp

```diff
@@ -212,7 +217,9 @@ ctkDICOMBrowserPrivate::ctkDICOMBrowserPrivate(ctkDICOMBrowser* parent, QSharedP
   : q_ptr(parent)
   , ImportDialog(0)
   , MetadataDialog(0)
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.